tests/tracking/fluent/test_fluent.py

```diff
@@ -62,20 +64,20 @@ def __init__(self):
 
     @classmethod
     def assert_values(cls, exp_id, name):
-        assert os.environ.get(_EXPERIMENT_NAME_ENV_VAR) == name
-        assert os.environ.get(_EXPERIMENT_ID_ENV_VAR) == exp_id
+        assert MLFLOW_EXPERIMENT_NAME.get() == name
```

Comment:
Is it possible to remove `HelperEnv` in a follow-up PR?

Reply:
We can directly use:

```python
assert MLFLOW_EXPERIMENT_NAME.get() == name
assert MLFLOW_EXPERIMENT_ID.get() == exp_id
```

and:

```python
monkeypatch.{setenv|devenv}
```

Reply:
monkeypatch does sound like a better option, I'll create a follow up PR